src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/WebRequestSession.cs

```diff
@@ -13,6 +14,20 @@ namespace Microsoft.PowerShell.Commands
     /// </summary>
     public class WebRequestSession
     {
+        private bool _changed;
```

Comment:
We can remove the field and use _httpClient field instead.
If we will create HttpClient in GetHttpClient() method this assumes the HttpClient will be started and can not be changed, if any parameter will be really changed, the fields should be set to null.